### PR TITLE
Add FFmpeg worker

### PR DESCRIPTION
**Pull request by valnoel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,44 @@
...
+      "paths": [
+        "/path/to/required/file"
+      ]
+    },
+    "input": {
+      "paths": [
+        "/path/to/source/file"
+      ]
+    },
+    "output": {
```
Comment (MarcAntoine-Arnaud) [inline README.md]: output can be an array, so you have to update with:
```json
"outputs" : [
  {
    "path": "...",
    "options": {}
  }
]
```

remark: the generated command line need to set options before output path ;-)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,44 @@
...
+```json
+{
+  "job_id": "1",
+  "parameters": {
+    "requirements": {
+      "paths": [
+        "/path/to/required/file"
+      ]
+    },
+    "input": {
```
Comment (MarcAntoine-Arnaud) [inline README.md]: update model with:
```json
"inputs": [
  {
    "path": "...",
    "options": {}
  }
]
```

File: worker.cfg [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[amqp]
+username=guest
+password=guest
+vhost=/
+hostname=192.168.99.100
+port=5672
+
+[ffmpeg]
+bin_path=/usr/bin
+lib_path=/usr/lib
```
Comment (MarcAntoine-Arnaud) [inline worker.cfg:10]: what append if nothing is set here (default ffmpeg installation on a unix)

Comment (valnoel) [reply worker.cfg:10]: A `RuntimeError` is raised, since 1adbc18

File: README.md (hunk 1 above)
Comment (valnoel) [reply README.md]: :+1: See a46c208

File: README.md (hunk 2 above)
Comment (valnoel) [reply README.md]: :+1: See a46c208

**Pull request merged**